hyper/http20/connection.py

```diff
@@ -482,6 +497,9 @@ def _send_cb(self, frame, tolerate_peer_gone=False):
 
         data = frame.serialize()
 
+        if frame.body_len > FRAME_MAX_LEN:  # pragma: no cover
```

Comment:
I also don't think there's any good reason to `#pragma: no cover` this, we can hit it pretty directly to prove the case.